ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/engine/AbstractEngineNewPayload.java

```diff
@@ -600,20 +612,23 @@ protected ValidationResult<RpcErrorType> validateBlobs(
     return ValidationResult.valid();
   }
 
-  private boolean validateExcessBlobGas(
+  private Optional<BlobGas> validateExcessBlobGas(
```

Comment:
would be good to have unit tests for this changed method

Reply:
Added unit tests, thanks